docs/channel/README.md

```diff
@@ -21,9 +21,9 @@ The dataplane is designed in a way that it only communicates the control plane v
 ## Comparison with other KafkaChannel implementations
 
 You may find a comparison of this new KafkaChannel implementation with
-the [consolidated KafkaChannel](https://github.com/knative-sandbox/eventing-kafka/blob/main/pkg/channel/consolidated/README.md)
+the [consolidated KafkaChannel]
```

Comment:
we are not releasing those anymore.

Can we close this PR?

Reply:
@matzew Did you mean we are not releasing these feature anymore or not releasing this doc anymore?

If we are not releasing these features anymore, do we need to change this README.md accordingly to reflect this?

If we are not releasing this doc anymore, I will close this PR then

Reply:
@Leo6Leo ok.
Let's update:
* point to the last release
* say that these _other_ channels are NOT supported 

OR even better: update the doc, and completely remove any mention of those unsupported other channel implementations. Makes sense?

Reply:
agreed! How about this we first update the doc to highlight the latest release and clarify that the other channels are not supported. After a transitional period, we'll fully remove mentions of these unsupported channels. I'll create an issue to track this. @matzew